src/main/js/store/actions/session.js

```diff
@@ -1,14 +1,5 @@
 import { push } from 'react-router-redux';
-import {
-    SESSION_CANCEL_LOGOUT,
-    SESSION_CONFIRM_LOGOUT,
-    SESSION_INITIALIZING,
-    SESSION_LOGGED_IN,
-    SESSION_LOGGED_OUT,
-    SESSION_LOGGING_IN,
-    SESSION_LOGIN_ERROR,
-    SESSION_LOGOUT
-} from 'store/actions/types';
+import { SAML_ENABLED, SESSION_CANCEL_LOGOUT, SESSION_CONFIRM_LOGOUT, SESSION_INITIALIZING, SESSION_LOGGED_IN, SESSION_LOGGED_OUT, SESSION_LOGGING_IN, SESSION_LOGIN_ERROR, SESSION_LOGOUT } from 'store/actions/types';
```

Comment:
Do we want this collapsed like this or expanded as it previously was? It is a lot easier to read all these if they're expanded in my opinion.